subprojects/libnvme.wrap

```diff
@@ -1,6 +1,6 @@
 [wrap-git]
 url = https://github.com/linux-nvme/libnvme.git
-revision = 501347ffdf6321d42ceabf173792c95bd44dac0d
+revision = 677075a11e8c2755d5566174c8321812f886ab29
```

Comment:
Could you split the libnvme.wrap update into a separate commit? So first update of the dependency and then do the change?

I'd like to avoid mixing 'CI' updates from code updates. Reason being for anyone doing backports it is a hassle when you need to pick a fix/change which also updates a file like libnvme.wrap. It will almost certainly always conflict in this case.

Reply:
ok, done!